api/src/Controller/PreRegistrationController.php

```diff
@@ -91,4 +93,11 @@ public function clientHasCoDeputies(string $caseNumber)
     {
         return $this->preRegistrationVerificationService->isMultiDeputyCase($caseNumber);
     }
+
+    private function replaceUnicodeChars(array $clientData)
```

Comment:
Might be better to name this `normaliseCharacters` or something instead as it's not a blanket replace of all unicode, just some characters we know get confused across the system.